mifosng-android/src/main/java/com/mifos/mifosxdroid/online/checkerinbox/CheckerInboxTasksFragment.kt

```diff
@@ -4,81 +4,39 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.lifecycle.Observer
-import androidx.lifecycle.ViewModelProviders
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.ui.platform.ComposeView
 import androidx.navigation.fragment.findNavController
-import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-import com.mifos.core.objects.checkerinboxandtasks.CheckerTask
 import com.mifos.mifosxdroid.R
 import com.mifos.mifosxdroid.core.MifosBaseFragment
-import com.mifos.mifosxdroid.core.util.Toaster
-import com.mifos.mifosxdroid.databinding.CheckerInboxTasksFragmentBinding
+import com.sparklead.feature.checker_inbox_task.checker_inbox_and_task.presentation.CheckerInboxTasksScreen
 import dagger.hilt.android.AndroidEntryPoint
-import javax.inject.Inject
 
 @AndroidEntryPoint
-class CheckerInboxTasksFragment : MifosBaseFragment(), SwipeRefreshLayout.OnRefreshListener {
-
-    private lateinit var binding: CheckerInboxTasksFragmentBinding
-
-    override fun onRefresh() {
-        viewModel.loadCheckerTasks()
-        viewModel.loadRescheduleLoanTasks()
-    }
-
-    companion object {
-        fun newInstance() = CheckerInboxTasksFragment()
-    }
-
-    @Inject
-    lateinit var factory: CheckerInboxViewModelFactory
-    private lateinit var viewModel: CheckerInboxTasksViewModel
-    private lateinit var swipeRefreshLayout: SwipeRefreshLayout
+class CheckerInboxTasksFragment : MifosBaseFragment() {
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        binding = CheckerInboxTasksFragmentBinding.inflate(inflater, container, false)
-        setToolbarTitle(resources.getString(R.string.checker_inbox_and_pending_tasks))
-        swipeRefreshLayout = binding.swipeRefreshLayout
-        swipeRefreshLayout.setOnRefreshListener(this)
-        return binding.root
-    }
-
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
-        showMifosProgressBar()
-        viewModel = ViewModelProviders.of(
-            this, factory
-        ).get(CheckerInboxTasksViewModel::class.java)
-
-        viewModel.getCheckerTasks().observe(viewLifecycleOwner,
```

Comment:
I don't see this initial load of getCheckerTasks() and getRescheduleLoanTasks() in composable screen

Reply:
I have created one GetCheckerInboxBadgesUseCase which consist both.